### PR TITLE
Update mistral version to st2-1.0.0

### PR DESCRIPTION
**Pull request by m4dcoder**
Update mistral version for the st2express and st2enterprise roles to st2-1.0.0.

### CODE REVIEW

File: hieradata/role/st2enterprise.yaml [hunk 1]
```diff
@@ -3,9 +3,9 @@ classes:
   - 'role::st2enterprise'
 
 st2::version: 0.14dev
-st2::revision: 132
+st2::revision: 133
```
Comment (jfryman) [inline hieradata/role/st2enterprise.yaml]: 154 as well?

Comment (m4dcoder) [inline hieradata/role/st2enterprise.yaml]: Are the revisions suppose to match between the two roles?

**Comment by m4dcoder** [comment]
Ok. revision updated to 154.

**Comment by manasdk** [comment]
+1

**Pull request merged**